src/Components/UI/Navigation/Navigation.jsx

```diff
@@ -0,0 +1,41 @@
+import { NavLink, Link } from "react-router-dom";
+import { useState } from "react";
+import "./Navigation.css";
+import logo from "../../../Assets/logo.png";
+
+export const Navigation = ({ verticalNavOpen, setVerticalNavOpen }) => {
+  const burgerRotate = verticalNavOpen ? "rotate" : "";
+
+  return (
+    <nav className={`navigation`}>
+      <NavLink to="/">
+        <img src={logo} alt="logo" className="img-responsive logo" />
+      </NavLink>
+
+      <form className="nav-search">
+        <input
+          type="search"
+          placeholder="Search for videos"
+          className="input-styled"
+        />
+        <button className="btn-icon">
+          <i className="fa fa-search" aria-hidden="true"></i>
+        </button>
+      </form>
+
+      <div
+        className={`burger ${burgerRotate}`}
+        onClick={() => {
+          setVerticalNavOpen((prevState) => !prevState);
+        }}
+      >
+        <div className="line line1"></div>
+        <div className="line line2"></div>
+        <div className="line line3"></div>
+      </div>
+    </nav>
+  );
+};
+
+//https://www.youtube.com/watch?v=uFgcSjVRJhE
+//https://www.youtube.com/watch?v=1c2-yWp-9pk
```

Comment:
Even if you are using these links for later. It would not make sense to have them in Navbar.